test/Feature/MaximalReconvergence/loop_peeling.test

```diff
@@ -0,0 +1,48 @@
+#--- source.hlsl
+RWStructuredBuffer<uint> Out : register(u0);
+
+[numthreads(8,1,1)]
+void main(uint3 TID : SV_GroupThreadID) {
+    for (uint i = 0; i < 8; i++) {
+        if (i == TID.x) {
+            Out[TID.x] = WaveActiveMax(TID.x);
+            break;
+        }
+    }
+}
+
+//--- pipeline.yaml
+
+---
+Shaders:
+  - Stage: Compute
+    Entry: main
+    DispatchSize: [1, 1, 1]
+Buffers:
+  - Name: Out
+    Format: UInt32
+    Data: [ 0, 0, 0, 0, 0, 0, 0, 0 ]
+DescriptorSets:
+  - Resources:
+    - Name: Out
+      Kind: RWStructuredBuffer
+      DirectXBinding:
+        Register: 0
+        Space: 0
+      VulkanBinding:
+        Binding: 0
+...
+
+#--- end
+# UNSUPPORTED: !VK_KHR_shader_maximal_reconvergence
+
+# BUG: https://github.com/llvm/llvm-project/issues/164496
+# XFAIL: Clang
```

Comment:
This bug should only affect Vulkan. The convergece tokens are not generated for DirectX.